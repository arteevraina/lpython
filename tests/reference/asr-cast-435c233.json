{
    "basename": "asr-cast-435c233",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/cast.py",
    "infile_hash": "6e4219b25f4aa2035b57c9658dbb133f2dff55db0d9c128b19214f62",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-cast-435c233.stderr",
    "stderr_hash": "94b3ddb64aab37aa358cd260252df21698826606401d00e07c873603",
    "returncode": 2
}