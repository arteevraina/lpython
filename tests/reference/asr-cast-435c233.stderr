warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
 --> tests/cast.py:5:9
  |
5 |     x = list(s)
  |         ^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
